lib/spack/spack/test/cmd/build_env.py

```diff
@@ -58,6 +63,31 @@ def test_pickle(tmpdir):
         assert "PATH" in environment
 
 
+# TODO params [i, b, c] require a spec that has proceeded with a directory
+# TODO praram [e] requires an active env
+@pytest.mark.parametrize("cd_key", ["r", "spack-root"])
+@pytest.mark.usefixtures("config", "mock_packages", "working_env")
+def test_cd(cd_key, tmpdir, monkeypatch, capfd):
+    """test that a subshell will navigate using spack cd before running commands"""
+    cmd = "pwd" if sys.platform != "win32" else "Get-Location"
```

Comment:
This line is causing the current CI failure

On posix systems `pwd` is a binary (or at least an executable script) available in the PATH. However on Windows, `Get-Location` is a powershell commandlet, meaning it has no meaning to the system if invoked via `exec` (or anything that isn't powershell). 

There's no good Windows equivalent of the `pwd` binary, you can either do some fancy work with direct win32 API calls, or just invoke a shell when executing the command (beware if it's powershell you'll need to post-process the value returned by `Get-Location` as it will be a powershell object). That could look something like:
```suggestion
    cmd = "pwd" if sys.platform != "win32" else 'powershell.exe -Command "& {(Get-Location).Path}'
```